CONTRIBUTING.md

```diff
@@ -1,33 +1,63 @@
+## Certificate of Origin
 
-The Conventional Commits specification is a lightweight convention on top of commit messages, which allow us to automate the release process. This convention dovetails with SemVer, by describing the features, fixes, and breaking changes made in commit messages. 
+By contributing to this project you agree to the Developer Certificate of
+Origin (DCO). This document was created by the Linux Kernel community and is a
+simple statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](DCO) file for details.
+
+## How to contribute:
+
+### Issues
+
+If you spot a problem with devfile alizer, [search if an issue already exists](https://github.com/devfile/api/issues). If a related issue doesn't exist, you can open a new issue using a relevant [issue form](https://github.com/devfile/api/issues/new/choose).
+
+You can tag Alizer related issues with the `/area alizer` text in your issue.
+
+### Development
+
+#### Repository Format
+
+The `alizer` repository includes different components:
+
+- [CLI](./README.md#cli)
+- [Alizer Library](./README.md#library-package)
+
+As a result `alizer` can be used both as a cli tool and imported as a package inside other projects. More information for the repository can be found [here](./docs/public/alizer-spec.md)
+
+#### Building locally
 
-Your commit messages should be structured as follows:
+More information for building & running locally the project can be found [here](./README.md#usage)
+
+#### Testing
+
+Apart from testing your changes locally with an updated Alizer CLI, someone can test their changes by running `make test`. This will test the updates against all existing test cases.
+
+### Submitting Pull Request
+
+**Note:** All commits must be signed off with the footer:
 
 ```
-<type>[optional scope]: <description>
+Signed-off-by: First Lastname <email@email.com>
 ```
 
-Read more about [commit types](https://github.com/pvdlg/conventional-commit-types#commit-types).
-### Certificate of Origin
+You can easily add this footer to your commits by adding `-s` when running `git commit`. When you think the code is ready for review, create a pull request and link the issue associated with it.
 
-By contributing to this project you agree to the Developer Certificate of
-Origin (DCO). This document was created by the Linux Kernel community and is a
-simple statement that you, as a contributor, have the legal right to make the
-contribution. See the [DCO](DCO) file for details.
+Owners of the repository will watch out for and review new PRs.
+
+By default for each change in the PR, GitHub Actions and OpenShift CI will run checks against your changes (linting, unit testing, and integration tests).
+
+If comments have been given in a review, they have to be addressed before merging.
+
+After addressing review comments, don't forget to add a comment in the PR afterward, so everyone gets notified by Github and know to re-review.
```

Comment:
@thepetk I'm thinking the author should mention the reviewer in their comment so the reviewer can provide a follow up review, wdyt?

```suggestion
After addressing review comments, don't forget to add a comment in the PR with the reviewer mentioned afterward, so they get notified by Github to provide a re-review.
```